src/views/LoanDetails/LoanDetails.js

```diff
@@ -298,23 +299,56 @@ class LoanDetails extends React.Component {
 
   showTitle(loan) {
     this.loan = loan;
-    const title = `${get(this.loan, ['item', 'title'], '')}`;
+    const title = get(this.loan, ['item', 'title'], '');
+    const instanceId = get(this.loan, ['item', 'instanceId'], '');
+    const holdingsRecordId = get(this.loan, ['item', 'holdingsRecordId'], '');
+    const isVirtualItem = isDCBItem({ instanceId, holdingsRecordId });
+
     if (title) {
       const titleTodisplay = (title.length >= 77) ? `${title.substring(0, 77)}...` : title;
```

Comment:
I know you didn't add this, but while you're here, can you move this magic number to a constant?